pulp_2to3_migration/app/migration.py

```diff
@@ -52,6 +52,8 @@ def migrate_repositories(plan):
     )
     with ProgressReport(**progress_data) as pb:
         for plugin in plan.get_plugin_plans():
+
+            # TODO: Should we filter out repos which have is_migrated=True?
```

Comment:
Do these TODO's need to be addressed before merging?

Reply:
I'd prefer to but it's not necessary. They don't affect current implementation, it will work without them.
I just noticed few places where I was not sure why we are not checking the is_migrated and I wanted to run it by you and @ipanova

Reply:
i think it should be the opposite, we should filter for is_migrated=True

Reply:
yes, let's set it to True, this way we will avoid additional step for repo_version_creation

Reply:
Discussed that we don't need that, the create_repo_version doesn't do any extra work if a repo is migrated but it ensures that all remotes are set properly.